src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -807,7 +825,11 @@ def storeTemplate(self, positionNumber, charBufferNumber = 0x01):
 
         ## DEBUG: Template stored successful
         if ( receivedPacketPayload[0] == FINGERPRINT_OK ):
-            return True
+            ## Return index when positionNumber was -1
+            if ( found == True ):
```

Comment:
But this feature should fix the issue. We should put the fix in the next stable release. We can't avoid that the new API breaks existing installations. If the users upgrade, they need to consider the notes in the changelog.